.github/workflows/release.yml

```diff
@@ -89,6 +103,17 @@ jobs:
       - name: Push artifacts to Box
         if: ${{ inputs.box_upload == 'true' }}
         run: |
-          echo "Do something"
-          
+          python3 promote-release-to-box.py \
```

Comment:
You're not using the `--jf_cli_rt_name` argument here. If I, in the `release.yml` workflow, change the name of [Artifactory config](https://github.com/pentaho/actions-common/pull/1/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34L63), inside the `promote-release-to-box.py` script another config will be added with the name `artifactory` pointing to the same instance.

Reply:
You're right I forgot to add jf cli, I've added it and rename the github summary echo

Reply:
If you're taking the name of the config from an `input`, you need to use [here](https://github.com/pentaho/actions-common/pull/1/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R81) also.

Reply:
fixed